pulp_ansible/app/tasks/collections.py

```diff
@@ -705,46 +710,39 @@ async def _find_all_collections(self):
     async def _should_we_sync(self):
         """Check last synced metadata time."""
         msg = _("no_change: Checking if remote changed since last sync.")
-        noop = ProgressReport(message=msg, code="sync.no_change")
-        noop.state = TASK_STATES.COMPLETED
-        noop.save()
-
-        if not self.repository.remote:
-            return True
```

Comment:
Same here, I don't know if the refactor was for other reasons but if it was just to resolve this issue you could do something like `await sync_to_async(self.repository.remote)` (might need to be wrapped in a function?)

Reply:
I decided to move it out of the async code